cassandane/Cassandane/Cyrus/Prometheus.pm

```diff
@@ -146,17 +146,36 @@ sub test_service_reportfile_exists
     # do something that'll get counted
     my $imaptalk = $self->{store}->get_client();
     $imaptalk->select("INBOX");
-    # and wait for a fresh report
-    sleep 3;
 
     my $fname = "$self->{instance}->{basedir}/conf/stats/service.txt";
+    my $last_timestamp = 0;
+    my $last_value = undef;
 
-    $self->assert_file_test($fname, '-f');
+    # get a few consecutive reports
+    for (1..3) {
```

Comment:
Should this test be marked `_slow`?

Reply:
It's only 9 seconds, so I don't think so.  Didn't bother me in individual testing, and disappears into the results in a larger run.  But I dunno, maybe?

If we think 9 seconds is too long for a non-slow test, then I'll undo the changes to the two existing tests, and put the timestamp checks into their own test which can be marked slow.

Reply:
For me 9 sec when run in bulk doesn't matter